pkg/apis/core/types.go

```diff
@@ -4771,9 +4771,9 @@ type ResourceQuotaScope string
 
 // These are valid values for resource quota spec
 const (
-	// Match all pod objects where spec.activeDeadlineSeconds
+	// Match all pod objects where `spec.activeDeadlineSeconds` >=0
```

Comment:
why adding backtick?

Reply:
Hmm I saw it in the example solution in the issue.

While grepping around I couldn't really see any uses like this though. - Happy to remove it.

Reply:
Removed them and squashed down to one commit.